### PR TITLE
add .drop = TRUE to unnest functions

### PR DESCRIPTION
**Pull request by double-y**
### Description
fix https://github.com/exploratory-io/tam/issues/5854

Test
- [x] Run functions that have tidyr::unnest in exploratory

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [x] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

**Comment by double-y** [comment]
Run devtools::check() and passed

File: R/util.R [hunk 1]
```diff
@@ -1018,3 +1018,21 @@ na_count <- function(x){
 na_pct <- function(x){
   sum(is.na(x)) / length(x) * 100
 }
+
+#' This is a wrapper of tidyr::unnest
+#' to change the default of .drop,
```
Comment (hideaki) [inline R/util.R]: Please explain the default behavior of tidyr::unnest.

File: R/arules.R [hunk 2]
```diff
@@ -95,7 +95,7 @@ do_apriori <- function(df, subject, key, minlen=1, maxlen=10, min_support=0.1, m
     stop("No matching rule was found.")
   }
 
-  ret <- (ret %>%  tidyr::unnest_(cnames[[5]]))
+  ret <- (ret %>%  unnest_with_drop_(cnames[[5]], .drop = TRUE))
```
Comment (hideaki) [inline R/arules.R]: Is .drop = TRUE necessary here?

File: R/util.R (hunk 1 above)
Comment (double-y) [inline R/util.R]: ok, added the explanation

File: R/arules.R (hunk 2 above)
Comment (double-y) [inline R/arules.R]: RIght, not necessary. removed it

File: R/util.R [hunk 3]
```diff
@@ -1018,3 +1018,29 @@ na_count <- function(x){
...
+#' so that it always drops other list
+#' columns, which is an expected behaviour
+#' for usage in this package in most cases.
+#' By default, unnest will drop other list columns
+#' if unnesting the specified columns requires the
+#' rows to be duplicated becouse of more than
```
Comment (hideaki) [inline R/util.R]: typo: becouse -> because

File: R/util.R [hunk 4]
```diff
@@ -1018,3 +1018,29 @@ na_count <- function(x){
...
+}
+
+#' This is a wrapper of tidyr::unnest
+#' to change the default of .drop,
+#' so that it always drops other list
+#' columns, which is an expected behaviour
+#' for usage in this package in most cases.
+#' By default, unnest will drop other list columns
+#' if unnesting the specified columns requires the
+#' rows to be duplicated becouse of more than
```
Comment (hideaki) [inline R/util.R]: typo: becouse

File: R/util.R (hunk 3 above)
Comment (double-y) [inline R/util.R]: Oh, right. fixed it

**Pull request merged**